processor/processor.go

```diff
@@ -916,10 +1163,59 @@ func (proc *HandleT) mainLoop() {
 	}
 }
 
-func (proc *HandleT) crashRecover() {
+func (proc *HandleT) readErrJobsLoop() {
+	logger.Info("Processor errors backup loop started")
 
 	for {
-		execList := proc.gatewayDB.GetExecuting([]string{gateway.CustomVal}, dbReadBatchSize, nil)
+		time.Sleep(errReadLoopSleep)
+		proc.statErrDBR.Start()
+
+		toQuery := errDBReadBatchSize
+		retryList := proc.errorDB.GetToRetry(nil, toQuery, nil)
```

Comment:
Usually we have jobsdb.Get{Unprocessed, GetToRetry...} etc one for a process.
gw is read, only by processor
batch_rt is read, only by batchrouter
rt is read, only by router etc.

How about having a separate error_processor or something like that that to read from errorDB and let those jobs be backed by this separate process instead of our regular processor.

Reply:
Yeah, this sounds good. We should move it to a separate file doing this.